packages/scanner/src/types.ts

```diff
@@ -0,0 +1,75 @@
+import { ProviderInterface } from '@polkadot/rpc-provider/types';
+import { Registry } from '@polkadot/types/types';
+import Metadata from '@polkadot/metadata/Decorated';
+
+export type RpcProvider = ProviderInterface;
+
+export type Hex = string;
+export type Bytes = string;
```

Comment:
@aniiantt what about using `AnyU8a` instead of `Bytes`
`import { AnyU8a } from '@polkadot/types/types';`

Reply:
Here I just need an alias for string, but AnyU8a is not string.